api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -86,9 +88,9 @@ public class FireCloudServiceImpl implements FireCloudService {
   private final Provider<ConvertApi> endUserStaticConvertApiProvider;
 
   private final Provider<WorkspacesApi> endUserWorkspacesApiProvider;
+  private final Provider<WorkspacesApi> endUserLenientTimeoutWorkspacesApiProvider;
```

Comment:
here I take the approach of keeping the special lenient **Provider**, but I don't need a special lenient **ApiClient** anymore - see the removal of this client inRawlsConfig